app/helpers/ems_cloud_helper/textual_summary.rb

```diff
@@ -26,6 +26,8 @@ def textual_group_smart_management
   end
 
   def textual_group_topology
+    items = %w(topology)
+    items.collect { |m| send("textual_#{m}") }.flatten.compact
```

Comment:
This should work equally well if the whole function was just `%i(topology)`. (Because `expand_textual_summary` already handles that case.)

But, it's pattern introduced by the other topologies and should be fixed everywhere .. separate PR .. will you or should I? :)